Add various thoughts (#1)
thomas-fossati/draft-fossati-cose-profile: draft-fossati-cose-profiles.md

    * using an OID {{-oid}}, URI {{-uri}} or CRI {{-cri}}
    * using a UUID {{-uuid}}
  * The chosen value SHOULD be appropriate for the intended usage scope (e.g., a small value when used in constrained node environments)

> * MAY define its own CBOR tag

Reviewer: we should probably get even more specific here (in later iterations)

Reviewer 2: would you mind raising an issue to track future actions?

> 
> This document has no IANA actions.
> 

Reviewer: ```suggestion
```

> Actually, it defines a header parameter "COSE_profile" that needs to

Reviewer: ```suggestion
This memo defines a header parameter "cose-profile" that needs to
```

Reviewer: Anticipating #2, I am changing this here already, too.

be registered in the "COSE Header Parameters" registry.

(Note that parameter names usually don't have programming language
lexicals, i.e., no underscores etc.)

> 

Reviewer: worth replacing right away

Reviewer 2: I see you've done that in #2
